libminifi/src/FlowController.cpp

```diff
@@ -175,7 +211,7 @@ void FlowController::stop(bool force) {
     this->flow_file_repo_->stop();
     this->provenance_repo_->stop();
     // Wait for sometime for thread stop
-    std::this_thread::sleep_for(std::chrono::milliseconds(1000));
+    std::this_thread::sleep_for(std::chrono::milliseconds(3000));
```

Comment:
it was the max of all thread sleep interval. for example, the repo sleep is 2 second